sdk/resourcemanager/azure-resourcemanager-storage/src/test/java/com/azure/resourcemanager/storage/StorageAccountOperationsTests.java

```diff
@@ -144,21 +146,23 @@ public void storageAccountDefault() {
             .define(saName)
             .withRegion(Region.US_EAST)
             .withNewResourceGroup(rgName)
+            .disableSharedKeyAccess()
             .create();
 
         Assertions.assertEquals(Kind.STORAGE_V2, storageAccountDefault.kind());
         Assertions.assertEquals(SkuName.STANDARD_RAGRS, storageAccountDefault.skuType().name());
         Assertions.assertTrue(storageAccountDefault.isHttpsTrafficOnly());
         Assertions.assertEquals(MinimumTlsVersion.TLS1_2, storageAccountDefault.minimumTlsVersion());
-        Assertions.assertTrue(storageAccountDefault.isBlobPublicAccessAllowed());
-        Assertions.assertTrue(storageAccountDefault.isSharedKeyAccessAllowed());
+        Assertions.assertFalse(storageAccountDefault.isBlobPublicAccessAllowed());
```

Comment:
`blobPublicAccessAllowed` by default is `false` now, and we don't seem to be able to change it, see [issue](https://github.com/Azure/azure-rest-api-specs/issues/35499)

Reply:
I can't find this `AllowBlobAnonymousAccess ` property in Swagger, or in created resource properties...
https://github.com/Azure/azure-rest-api-specs/blob/main/specification/storage/resource-manager/Microsoft.Storage/stable/2024-01-01/storage.json